corehq/apps/export/models/new.py

```diff
@@ -1607,6 +1698,36 @@ def _add_to_group_schema(group_schema, path_start, prop, app_id, app_version):
         return schema
 
 
+class SMSExportDataSchema(ExportDataSchema):
+    @property
+    def type(self):
+        return SMS_EXPORT
+
+    @classmethod
+    def schema_version(cls):
+        return SMS_DATA_SCHEMA_VERSION
+
+    @classmethod
+    def _get_inferred_schema(cls, domain, identifier):
+        return None
+
+    @classmethod
+    def _get_current_app_ids_for_domain(cls, domain, app_id):
```

Comment:
I didn't like that these needed to be implemented here. They are called in the base classes generate schema class.

@benrudolph I was thinking that this framework could be pretty useful in the future if we want to move other reports to the export framework (like worker activity or whatever). What do you think about splitting up the base classes into a `ExportDataSchema` and `AppDependentExportDataSchema`? I wasn't sure how non app dependent exports should fit into this framework